tests/integration/test_graphql_query.py

```
@@ -0,0 +1,172 @@
+# type: ignore
+from django.contrib.auth import get_user_model
+from django.db.models import CharField, IntegerField, DateField, ForeignKey, CASCADE
```

Ruff (0.12.2)

`django.db.models.IntegerField` imported but unused

Remove unused import: `django.db.models.IntegerField`

(F401)